File: Course_01/C++ Language/Matrizes/soma_matrizes.cpp
#include <iostream>

using namespace std;

int main()
{
    int m, n, i, j;

    cout << "Qual a quantidade de linhas da matriz? ";
    cin >> m;
    cout << "Qual a quantidade de colunas da matriz? ";
    cin >> n;

    int matA[m][n], matB[m][n], matSoma[m][n];

    cout << "Digite os valores da matriz A: " << endl;
    for (i = 0; i < m; i++) {
        for (j = 0; j < n; j++) {
            cout << "Elemento [" << i << "," << j << "]: ";
            cin >> matA[i][j];
        }
    }
    cout << "Digite os valores da matriz B: " << endl;
    for (i = 0; i < m; i++) {
        for (j = 0; j < n; j++) {
            cout << "Elemento [" << i << "," << j << "]: ";
            cin >> matB[i][j];
        }
    }

    for (i = 0; i < m; i++) {
        for (j = 0; j < n; j++) {
            matSoma[i][j] = matA[i][j] + matB[i][j];
        }
    }

    cout << "MATRIZ SOMA: " << endl;

    for (i = 0; i < m; i++) {
        for (j = 0; j < n; j++) {
            cout << matSoma[i][j] << " ";
        }
        cout << endl;
    }

    return 0;
}

File: Course_01/Python/Matrizes/soma_matrizes.py
m = int(input("Qual a quantidade de linhas da matriz? "))
n = int(input("Qual a quantidade de colunas da matriz? "))

print("Digite os valores da matriz A:")
matrizA = [[0 for x in range(n)] for x in range(m)]

for i in range(0, m):
    for j in range(0, n):
        matrizA[i][j] = int(input(f"Elemento [{i},{j}]: "))

print("Digite os valores da matriz B:")
matrizB = [[0 for x in range(n)] for x in range(m)]

for i in range(0, m):
    for j in range(0, n):
        matrizB[i][j] = int(input(f"Elemento [{i},{j}]: "))

matrizSoma = [[0 for x in range(n)] for x in range(m)]

for i in range(0, m):
    for j in range(0, n):
        matrizSoma[i][j] = matrizA[i][j] + matrizB[i][j]

print("Matriz soma:")

for i in range(0, m):
    for j in range(0, n):
        print(f"{matrizSoma[i][j]} ", end="")
    print()